app/views/admin_mailer/daily_uploads_report.text.erb

```diff
@@ -0,0 +1,13 @@
+<% @data_imports.each do |data_import| %>
+  <%= "#{data_import.occupation_standard.title} (Public link)" %>: <%= occupation_standard_url(data_import.occupation_standard, host: ENV.fetch("PUBLIC_DOMAIN", Rails.application.config.action_mailer.default_url_options[:host])) %>
+  <%= "#{data_import.occupation_standard.title} (Admin link)" %>: <%= admin_occupation_standard_url(data_import.occupation_standard) %>
+  Competencies count: <%= data_import.occupation_standard.competencies_count %>
+  OJT hours min: <%= data_import.occupation_standard.ojt_hours_min %>
+  OJT hours max: <%= data_import.occupation_standard.ojt_hours_max %>
+  RSI hours min: <%= data_import.occupation_standard.rsi_hours_min %>
+  RSI hours max: <%= data_import.occupation_standard.rsi_hours_max %>
+  Link to Admin Data Import: <%= admin_data_import_url(data_import) %>
+  Link to Admin Source File: <%= admin_source_file_url(data_import.source_file) %>
+
+
```

Comment:
picky: couple of blank lines added

Reply:
These blank lines are on the text version so that there will be breaks between the imports listed (see screenshots). There are some corresponding `<br>` tags on the HTML version.